Covert to V1, update Build

rxm7706 opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->
dsnparse 0.1.15 is not supported on this platform
trying to publish https://pypi.org/project/collate-data-diff
which is needed by https://github.com/open-metadata/OpenMetadata/blob/1.8.0-release/ingestion/setup.py#L160

<!--
Please add any other relevant info below:
-->

rxm7706 committed:
Pixi

rxm7706 committed:
Covert to V1 recipe

rxm7706 committed:
Delete recipe/meta.yaml

rxm7706 commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

Commit:
MNT: Re-rendered with conda-build 25.5.0, conda-smithy 3.50.1, and conda-forge-pinning 2025.06.22.18.05.36

rxm7706 committed:
Add Maintainer

rxm7706 commented:
@astrojuanlu 
I need an older version of dnsparse -- dsnparse 0.1.15 for collate-data-dif 
I have updated the recipe to use V1 format, and added myself as a maintainer 
I create a second PR to merge v0.1.15 into a new 1.x branch to publish the older version I need. 
Thank you and Regards.

astrojuanlu reviewed (approved):
LGTM!

rxm7706 commented:
> LGTM!

Thanks @astrojuanlu ! 
Could you merge this when you get a chance.. 
Regards
